doc/Makefile.am

```diff
@@ -68,8 +68,10 @@ deprecated-clean:
 
 # Annotated source code as HTML
 
+# Cleaning first ensures we don't index unrelated stuff like RPM sources
 global:
 	$(MAKE) $(AM_MAKEFLAGS) -C .. clean-generic
+	$(MAKE) $(AM_MAKEFLAGS) -C ../rpm rpm-clean
 	cd .. && gtags -q && htags -sanhIT doc
```

Comment:
It might be worth adding the global package as an optional build dependency in INSTALL.md, then.  I didn't know about this until just now.